R/check-cardinalities.R

```diff
@@ -1,33 +1,33 @@
 #' Test if the relation between two tables of a data model meet the requirements
 #'
-#' @description All `check_cardinality()` functions test, if the `pk_column` is a unique key for `parent_table` and if
+#' @description All `check_cardinality()` functions test if `pk_column` is a unique key for `parent_table` and if
```

Comment:
```suggestion
#' @description All `check_cardinality()` functions test the following conditions:
#' 1. Is `pk_column` is a unique key for `parent_table`?
#' 1. Is the set of values in `fk_column` of `child_table` a subset of the set of values of `pk_column`?
#' 1. Does the relation between the two tables of the data model meet the cardinality requirements ?
#`
#` Please see below for details.
```

Reply:
I tried to create a multi-line suggestion here but the conversation comment was between the lines.

Reply:
Can you please make these changes in your clone and push? The updated version should appear here.